.github/workflows/Docker-image-to-package.yml

```diff
@@ -0,0 +1,43 @@
+name: Create and publish a Docker image to ghcr.io
+
+on:
+  push:
+     branches: [ "main" ]
+  pull_request:
+     branches: [ "main" ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
```

Comment:
What is the hash value after login-action ?

Reply:
The hash refers to specific version of docker/login-action. Here, it refers to v2. The hash can be replaced with "docker/login-action@v2" at every instance of the codes. For more reference, [you can refer here.](https://github.com/Pradumnasaraf/SocialSlash/issues/17)